readme.md

```diff
@@ -819,6 +820,92 @@ Type: `string`
 
 The IP address used to send the request from.
 
+### HTTPS
```

Comment:
I don't think it's clear enough what this means. Does it mean that these options will be ignored when not using HTTPS or will it throw an error.

Reply:
```suggestion
### Advanced HTTPS API
```

Reply:
> Does it mean that these options will be ignored when not using HTTPS or will it throw an error.

I think it should just ignore them.
I can write a line right after the "HTTPS" (or whatever we are going to call the section) title like:
`Note: If the request is not HTTPS these options will be ignored.`